### ISSUE TITLE
OMERO-py (Java in the background?) can not reload annotations

### ISSUE THREAD

**Issue by stefanches7** [ome/omero-py#423]
clone of ome/omero-gateway-java#88

**Comment by stefanches7** [comment]
https://forum.image.sc/t/metadata-access-with-omero-matlab-language-bindings-fails/37719

**Comment by joshmoore** [comment]
I missed this, somehow, @stefanches7. Did you work things out?

**Comment by stefanches7** [comment]
> I missed this, somehow, [@stefanches7](https://github.com/stefanches7). Did you work things out?

Yup @joshmoore , I use `conn._loadAnnotationLinks()`. Not pretty, but it does the thing.
